### ISSUE TITLE
city added after pbd sensor

### ISSUE THREAD

**Issue by GitHubxtsnl** [xirixiz/homeassistant-afvalwijzer#229]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/xirixiz/homeassistant-afvalwijzer/blob/master/.github/SUPPORT.md)

### Description

All the sensors are displayed as "afvalwijzer + id + afvaltype".
in my case:
afvalwijzer RP39 papier --> sensor.afvalwijzer_rp39_papier
afvalwijzer RP39 gft --> sensor.afvalwijzer_rp39_gft
afvalwijzer RP39 textiel --> sensor.afvalwijzer_rp39_textiel
afvalwijzer RP39 kerstbomen --> sensor.afvalwijzer_rp39_kerstbomen

Except sensor.afvalwijzer_rp39_plastic_blik_drinkpakken this contains the suffix **_arnhem**
![image](https://user-images.githubusercontent.com/43134674/223550829-ba7a7abb-889e-413d-930d-325ed087a365.png)

Here is my code in configuration.yaml

```
# HACS - INTEGRATION FOR AFVALWIJZER
  - platform: afvalwijzer
    provider: prezero
    postal_code: !secret general_address_zipcode
    street_number: !secret general_address_number
    suffix: ''
    exclude_pickup_today: false
    default_label: Geen
    id: 'RP39'
    exclude_list: ''
```
I believe the suffix should not be there in the afvalwijzer_rp39_plastic_blik_drinkpakken sensor.

### Expected behaviour

I believe the suffix should not be there in the afvalwijzer_rp39_plastic_blik_drinkpakken sensor.

### Actual behaviour

The suffix "_arnhem" is added to the afvalwijzer_rp39_plastic_blik_drinkpakken sensor.

### Version

HomeAssistant 2023.3.1
Supervisor 2023.01.1
Operating System 9.5
Afvalwijzer 2023.03.05

### HASS logs

no logging made yet,

**Comment by xirixiz** [comment]
Interesting :). Can you share your postal code so I can perform some validations?

**Comment by GitHubxtsnl** [comment]
> Interesting :). Can you share your postal code so I can perform some validations?

Have you managed to find anything? Can I assist in testing/troubleshooting?

**Comment by xirixiz** [comment]
Fixed in release: 2023.04.02
